inst/rmarkdown/templates/thesis-paged/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: A paged html thesis template for R Markdown user's
+subtitle: A {pagedown} template.
+author: 
+  - name: William Brent Thorne
+    edu: BSc
+degree: Master of Documentation
+institute: Typeset University
+faculty: Reproducibility and FOSS
+department: Templates
+location: St. Catharines, ON 
+date:
+  - year: 2019
+    month: August
+sign_page: true
+dedication: |
+  `r paste(readLines("dedication.Rmd"), collapse = '\n  ')`
+abstract: | 
+  `r paste(readLines("abstract.Rmd"), collapse = '\n  ')`
+preface: |
+  `r paste(readLines("preface.Rmd"), collapse = '\n  ')`
+acknowledge: |
+  `r paste(readLines("acknowledge.Rmd"), collapse = '\n  ')`
+committee:
+  - name: Jane Doe III
+    prefix: Dr
+    position: Chair of Department
+  - name: John Smith
+    prefix: Dr
+    position: Faculty Advisor
+lof: true
+lot: true
+toc-title: Contents
+output:
+  pagedown::thesis_paged: 
+    toc: true
+    number_sections: yes
+    pandoc_args: --mathjax
+    self_contained: no
+bibliography: Thesis.bib
+link-citations: yes
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE,
+                      fig.align = 'center')
+```
+
+
+# Literature Review {-}
+
+This is where you can include a lit review if you don't whish for it to be an individual chapter or to be numbered. To make sure that a section heading is not numbered use the `{-}` notation beside the header text like this:
+
+_**rmarkdown**_:
+
+```markdown
+# Literature Review {-}
+
+This is where you can include a lit review if you don't whish for it to be an individual chapter or to be numbered. To make sure that a section heading is not numbered use the `{-}` notation beside the header text like this:
+```
+
+```{r btable}
+knitr::kable(iris[1:4,1:5], align = 'c',caption = 'A non nested table and its caption for the table.',col.names = c("Sepal L","Sepal W", "Petal L", "Petal W", "Species"))
+```
+
+```{r child = 'chap1.Rmd'}
```

Comment:
I understand why you need child documents here.

For the record, I tried to use `rmarkdown::render_site()` but the problem I faced is that `rmarkdown::render_site()` only produces the html files and does not start a local webserver. 
In other words, I didn't find in the **\*down** packages a function like `xaringan::inf_mr()` which would also work for a bookdown project.

Reply:
I've removed thesis_minimal and have condensed thesis_paged into a single rmd file which does not create a directory. I will put together an extensive example of a thesis using child docments and such on my github so people can refer to it if they wish to see how far they can take it without adding too much bloat to the actual template.